[IMP] create JavaScript howtos (#3969)
odoo/documentation: content/developer/howtos/javascript_client_action.rst

==========================
Create a client action
==========================

A client action triggers an action that is entirely implemented in the client side.
One of the benefits of using a client action is the ability to create highly customized interfaces

> with ease. A client action is typically defined by an OWL component, we can also use the web
> framework and use services, core components, hooks,...

Reviewer: ```suggestion
with ease. A client action is typically defined by an OWL component; we can also use the web
framework and use services, core components, hooks,...
```

> #. First we have to create the :ref:`client action <reference/actions/client>`, don't forget to
>    make it accessible

Reviewer: ```suggestion
#. Create the :ref:`client action <reference/actions/client>`; don't forget to
   make it accessible.
```

   .. code-block:: xml

      <record model="ir.actions.client" id="my_client_action">
          <field name="name">My Client Action</field>
          <field name="tag">my_module.MyClientAction</field>
      </record>

> #. Create a component that represents our client's action

Reviewer: ```suggestion
#. Create a component that represents the client action.
```

======================
Create a client action
======================

A client action triggers an action that is entirely implemented in the client side.
One of the benefits of using a client action is the ability to create highly customized interfaces
with ease. A client action is typically defined by an OWL component; we can also use the web
framework and use services, core components, hooks,...

> #. Create the :ref:`client action <reference/actions/client>`, don't forget to
>    make it accessible

Reviewer: ```suggestion
#. Create the :ref:`client action <reference/actions/client>`; don't forget to
   make it accessible.
```

> =============

Reviewer 2: the main title of this document should be "Create a new client action"

> Client action

Reviewer: ```suggestion
Create a client action
```
